docs/conf.py

```diff
@@ -48,7 +48,7 @@
 
 # General information about the project.
 project = u'Pillow (PIL Fork)'
-copyright = u'1995-2016, Fredrik Lundh and Contributors, Alex Clark and Contributors'
+copyright = u'1995-2017, Fredrik Lundh and Contributors, Alex Clark and Contributors'
```

Comment:
* PIL was born in 1995: https://github.com/python-pillow/Pillow/issues/1276

* Pillow was forked from PIL (1.1.7 final): https://github.com/python-pillow/Pillow/commit/9a640e3157150e35f17354517e8c6304fc428060#diff-e4eb329834da3d36278b1b7d943b3bc9

* PIL 1.1.7 was released November 15, 2009: http://www.pythonware.com/products/pil/

* Pillow was forked in July 2010: https://github.com/python-pillow/Pillow/milestone/13

* So, 1995-2009 and 2010-2017?